src/main/java/address/controller/RootLayoutController.java

```diff
@@ -119,6 +121,8 @@ private void handleSaveAs() {
         }
 
         modelManager.setPrefsSaveLocation(file.getPath());
+        // TODO temp patch to create cloud file if non found
+        raise(new CreateAddressBookOnRemoteRequestEvent(new CompletableFuture<>(), file.getName()));
```

Comment:
I though this should be done automatically by the cloud simulator? i.e. create the file if it doesn't exist